maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -190,7 +194,7 @@ private void deleteDistroFiles(File targetDir) {
 			new File(targetDir, OPENMRS_DISTRO_PROPERTIES).delete();
 		}
 		catch (IOException e) {
-			e.printStackTrace();
+			log.error(e.toString(), e);
```

Comment:
If you look at the rest of the codebase, we do not usually do `e.toString()`

Reply:
Isn't e.getMessage() better than e.toString()?

Reply:
Yeah, I'll change it.

Reply:
@dkayiwa I changed it.